### ISSUE TITLE
Create Redux Module for Flash Messages

### ISSUE THREAD

**Issue by lionize** [praelatus/frontend#135]
We need a way to control notifications, keep track of their viewed/unviewed state, and also keep track of their history in case the user wants to view previous messages.

**Comment by chasinglogic** [comment]
Would love for these to be more "Notifcation" esque, like a growler message on Mac OSX where it pops up in the top right and is dismissable, and multiple notifications will "stack up"

**Comment by lionize** [comment]
@chasinglogic Yeah that's what I was thinking. Something a lot more interactive and useful than just having a message you'll ignore at the top of the screen now and then.
